testdata.log

```diff
@@ -8412,23 +8412,17 @@ SELECT * FROM t WHERE  EXISTS ( SELECT * FROM t WHERE i == 2 ) ORDER BY i;
 
 ---- 1349
 SELECT * FROM t WHERE  EXISTS ( SELECT * FROM t WHERE i == 2 );
```

Comment:
`SELECT * FROM t WHERE  EXISTS ( SELECT * FROM t WHERE i == 2 );`

vs

```
┌Iterate no rows
└Output field names ["i"]
```

does not look correct. I assume #165, which I haven't looked at yet maybe fixes that. If not, this is not critical and can be resolved later. It will affect only the result of `EXPLAIN ...`.